src/main/java/frc/team2412/robot/util/auto/AutoLogic.java

```diff
@@ -474,7 +479,7 @@ public static Command feederIn() {
 		return (INTAKE_ENABLED && LAUNCHER_ENABLED
 						? Commands.waitUntil(isReadyToLaunch())
 								.andThen(Commands.waitSeconds(FEEDER_DELAY))
-								.andThen(new FeederInCommand(s.intakeSubsystem))
+								.andThen(new FeederInCommand(s.intakeSubsystem).until(untilNoNote())
```

Comment:
```suggestion
								.andThen(new FeederInCommand(s.intakeSubsystem).until(untilNoNote()))
```
Missing closing parentheses is causing the compile error.